templates/repo/issue/view_content/pull.tmpl

```diff
@@ -471,7 +472,7 @@
 				</div>
 			{{end}}
 
-			{{if and .ShowMergeInstructions .Issue.PullRequest.HeadRepo}}
+			{{if and .ShowMergeInstructions .Issue.PullRequest.HeadRepo (not .Issue.PullRequest.HasMerged)}}
```

Comment:
Hmm… only if it was merged?
Or also when the PR is closed?

Reply:
Hmm, I see a slight usefulness on having in on the close PR in case it gets re-opened. A merged PR can not be reopened to my knowledge.

Reply:
Thought I do see closed PR has a similar UI that needs similar treatment:

<img width="859" alt="image" src="https://user-images.githubusercontent.com/115237/230480410-867fa9f5-a74a-43ee-a4d2-4cbf9e28941b.png">